### PR TITLE
Worker Integration and test updates

### PR DESCRIPTION
**Pull request by deepesh-kn**
This PR includes the following:

**MOSA-26:** Interface workers
**MOSA-32:** Story request stake
**MOSA-33:** Story request stake reject
**MOSA-34:** Story request stake revert
**MOSA-35:** Story revert stake
**MOSA-36:** Story revert stake hunter (non whitelisted worker)
And Story Process staking

### CODE REVIEW

File: contracts/Workers.sol [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+import "./SafeMath.sol";
+import "./EIP20Interface.sol";
+import "./OpsManaged.sol";
+
+/// A set of authorised workers
+contract Workers is OpsManaged {
```
Comment (benjaminbollen) [inline contracts/Workers.sol]: `is WorkersInterface`

Comment (deepesh-kn) [reply contracts/Workers.sol]: Done !!!

File: contracts/Gate.sol [hunk 2]
```diff
@@ -84,7 +85,7 @@ contract Gate is ProtocolVersioned, Owned {
         require(_workers != address(0));
         require(_uuid.length != uint8(0));
 
-        workers = _workers;
+        workers = WorkersInterface(_workers);
```
[the remark this answers is not in the document]
Comment (deepesh-kn) [reply contracts/Gate.sol]: Done, changed parameter as WorkersInterface in L77

**Comment by benjaminbollen** [comment]
Thanks for the changes. Looks great

**Pull request merged**